xla/service/gpu/transforms/gemm_rewriter.cc

```diff
@@ -48,6 +48,7 @@ limitations under the License.
 #include "xla/layout.h"
 #include "xla/literal.h"
 #include "xla/literal_util.h"
+#include "xla/permutation_util.h"
```

Comment:
You need to update the BUILD file and add a dependency to the target that has this header in its hdrs section.

Reply:
Also missing:

#include "xla/layout_util.h"